packages/alert-message-react/src/common/MessageIcon.tsx

```diff
@@ -6,42 +6,68 @@ export function MessageIcon({ messageType }: { messageType: messageTypes }) {
                         fill="currentColor"
                     />
                 </svg>
             );
         case "info":
             return (
-                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-                    <circle cx="12" cy="12" r="11.5" stroke="currentColor" />
+                <svg
+                    className="jkl-alert-message__icon"
+                    aria-hidden
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                >
                     <path
-                        d="M11.952 7.328C12.384 7.328 12.688 7.072 12.688 6.624C12.688 6.192 12.384 5.92 11.952 5.92C11.536 5.92 11.248 6.192 11.248 6.624C11.248 7.072 11.536 7.328 11.952 7.328ZM11.504 18H12.512V9.408H11.504V18Z"
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M24 12C24 18.6274 18.8503 24 12 24C5.59548 24 0 18.6274 0 12C0 5.37258 5.59548 0 12 0C18.8503 0 24 5.37258 24 12ZM11 17.4142V10H13.2229V17.4142H11.2229ZM12 8.5C12.9133 8.5 13.4729 7.94036 13.4729 7.25C13.4729 6.55964 12.9133 6 12 6C11.5325 6 10.9729 6.55964 10.9729 7.25C10.9729 7.94036 11.5325 8.5 12 8.5Z"
                         fill="currentColor"
                     />
                 </svg>
             );
         case "success":
             return (
-                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-                    <circle cx="12" cy="12" r="11.5" stroke="currentColor" />
-                    <path d="M7 13.5L10 16.5L19.5 7" stroke="currentColor" />
+                <svg
+                    className="jkl-alert-message__icon"
+                    aria-hidden
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                >
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M12 24C18.8503 24 24 18.6274 24 12C24 5.37258 18.8503 0 12 0C5.59548 0 0 5.37258 0 12C0 18.6274 5.59548 24 12 24ZM11.5503 16.7071L19.0503 9.20711L17.6361 7.79289L10.8432 14.5858L7.26909 11.0116L5.85486 12.4258L10.1361 16.7071L10.8432 17.4142L11.5503 16.7071Z"
+                        fill="currentColor"
+                    />
                 </svg>
             );
         case "warning":
             return (
-                <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
-                    <circle cx="12" cy="12" r="11.5" stroke="currentColor" />
+                <svg
+                    className="jkl-alert-message__icon"
+                    aria-hidden
+                    viewBox="0 0 26 24"
```

Comment:
Virker som at dette litt bredere ikonet skaper en aldri så liten ulikhet mellom variantene hvis de ikke har noen tittel. Eksempel fra dokumentasjonen til `AlertMessage`:

https://user-images.githubusercontent.com/1223410/134463074-7034d4ab-fd27-4827-b384-30cbd87bf442.mov


Det ser helt supert ut med en tittel som i `MessageBox`. Er det en ekstramargin vi kan slenge på selve brødteksten for å få pushet pixlene på samme måte? 😄 Er vel heller usannsynlig at vi kommer til å vise flere av disse side om side slik at det synes, men om det er en quickfix så hjelper det på opplevelsen i portalen om ikke annet.

Reply:
Hehe, det tenkte jeg ikke på. Tror forskjellen er at den er absolutt posisjonert i `MessageBox`, men ikke i `AlertMessage` pga. litt annen layout/oppførsel. Skal se om jeg kan få laget en kjapp fiks på det! Fint å være litt perfeksjonist 😊